README.markdown

```diff
@@ -65,7 +65,8 @@ Example:
         "fontname": "Comic Sans MS",
         "fontsize": 30,
         "bgcolor": [20, 120, 20],
-        "textcolor": [180, 20, 180]
+        "textcolor": [180, 20, 180],
+        "cover_url": "https://website.com/image.png"
```

Comment:
It feels weird to put a specific image here in the defaults, which will apply to every story... but I guess I can see it for consistency.